bukkit/src/main/java/com/griefdefender/hooks/provider/DynmapProvider.java

```diff
@@ -100,11 +99,17 @@ private String getWindowInfo(Claim claim, AreaMarker marker) {
         info = info.replace("%claimname%",
                 claim.getData().getDisplayNameComponent().isPresent()
                         ? PlainComponentSerializer.plain().serialize(claim.getDisplayNameComponent().get())
-                        : "none");
+                        : cfg.missingNamePlaceholder);
         final Date lastActive = Date.from(claim.getData().getDateLastActive());
         info = info.replace("%lastseen%", lastActive.toString());
         info = info.replace("%gdtype%", claim.getType().toString());
 
+        info = info.replace("%width%", Integer.toString(claim.getWidth()));
+        info = info.replace("%length%", Integer.toString(claim.getLength()));
+        info = info.replace("%height%", Integer.toString(claim.getHeight()));
+        info = info.replace("%area%", Integer.toString(claim.getArea()));
```

Comment:
This is a duplicate area check. Remove the other.

Reply:
Duplicate placeholder has been removed.